### ISSUE TITLE
Viewing a docked playlist during video playback causes VLC to crash

### ISSUE THREAD

**Issue by esbeeb** [RPi-Distro/vlc#91]
I have an RPi4, 8GB of RAM, and a 1080p monitor. Raspberry Pi OS Bookworm 64bit, with default VLC, up-to-date as of this post.

I opened a 1080p 60fps vp9-encoded .mp4 video file in VLC, and it runs decently well, even in fullscreen.

But **during that playback** VLC instantly crashes if I want to "View" -> "Playlist" (and note that "Docked Playlist" is just under "Playlist", and has a checkmark by default).

_Workaround:_
If I first click that "Docked Playlist" to remove that checkmark, then click "Playlist", now the Playlist opens in its own window. No crash in that case: a sub-window for the Playlist opens as expected. It's just that (default) Docked Playlist which causes the crash.

**Comment by jc-kynesim** [comment]
Yup - that definitely happens - I'll look into it (though I'm guessing it'll turn out to be a Qt/Way* issue)

**Comment by jc-kynesim** [comment]
I know what causes this (qt deletes the surface that egl was drawing on) and the (new) default wayland output module should no longer have this issue in the next release.
